### ISSUE TITLE
[Suggestion] Ability to specify item priority when recipes use forge tags

### ISSUE THREAD

**Issue by Nubidubi23** [refinedmods/refinedstorage#3037]
I find it kinda annoying how i have 100k stone and 1000 andesite and recipes which require forge:stone automatically default to andesite. A list of some sort where priorities could be set for each tag would be great.

**Comment by 19175** [comment]
Another way of doing this (and the way AE2 does it I think) would be to use whatever there is most of in the system, so in a situation where you have 1000 stone and 100k andesite it would use the andesite first.

**Comment by raoulvdberge** [comment]
Yes, when allowing alternative tags in Refined Storage 2, the item that you have the most of is used first now.
